datahub-actions/src/datahub_actions/plugin/action/metadata_change_sync/metadata_change_sync.py

```diff
@@ -58,6 +58,9 @@ def __init__(self, config: MetadataChangeEmitterConfig, ctx: PipelineContext):
             self.aspects_exclude_set = self.DEFAULT_ASPECTS_EXCLUDE_SET.union(
                 set(self.config.aspects_to_exclude)
             )
+        logger.info(
+            f"MetadataChangeSyncAction configured to emit mcp to gms server {self.config.gms_server} with extra headers {self.config.extra_headers}"
```

Comment:
For logging the headers, can we just log the keys and not values because values might contain sensitive info?
Also can we log the excluded aspects?